### ISSUE TITLE
Problem with downloading image from ACR (specific password)

### ISSUE THREAD

**Issue by mkosieradzki** [Azure/service-fabric-issues#1466]
`!F+Pi^-|&D]?#*!5:v+I])i-2^!@|F[?@&}]l`

This password (generated by AAD) causes problems when logging on to ACR from Service Fabric to download container image. There must be some character causing problems here. My first assumption was that the problem was with escaping & - but that was not it, my second assumption was that the issue was cause by `[something]` combination that could introduce substitution, but I passed the argument using PowerShell to make sure it is not being substituted. So it must be something else. I have no idea which character is causing problems, but the ACR seem to respond with 500 during logon.

**Comment by ashishnegi** [comment]
@mkosieradzki Are you saying that SF is not able to download container images if password is like first string you have pasted ?

**Comment by mkosieradzki** [comment]
@ashishnegi yes. (apologies for delayed answer, I didn't get the notification)

**Comment by mkosieradzki** [comment]
One more password causing problems: `i]?Q+0U.aDbgzJ79PfhvJCwSl72AL/CM` So it is probably either `+`, `?`, `]` (but I have another password with `]` and it works). So the remaining candidates are: `+` and `?`.
